ComparisonWithOrleans.md

```diff
@@ -0,0 +1,203 @@
+# Orleans and Akka Actors: A Comparison
+
+## Step-By-Step Review
+
+This overview walks along the Technical Report [MSR-TR-2014-41](http://research.microsoft.com/apps/pubs/default.aspx?id=210931) by *Philip A. Bernstein, Sergey Bykov, Allan Geller, Cabriel Kliot, Jorgen Thelin (Microsoft Research)*, for another comparison focusing on Erlang and Riak see [Christopher Meiklejohn’s excellent article](http://christophermeiklejohn.com/orleans.html).
+
+### Introduction
+
+Different focus:
+
+  * Akka as a toolkit for building distributed systems, offering the full power but also exposing the inherent essential complexity.
+  
+  * Orleans restricts applicability in order to allow seamless use without understanding distributed computing.
+  
```

Comment:
There is one very important point that needs to be mentioned. From the very early stages of the runtime, at almost any decision we made, we always came from the standpoint of: first, what  should be the default behavior, what would be the most natural and easy for non expert. Second, how an expert can change the default, how it can have more flexibility.We tried not to sacrifice the 2nd.
You will see examples of this principle: simple first, extensibility second, in a lot of comments I make below.